test/Nancy.Tests/Unit/HttpLinkHeaderFixture.cs

```diff
@@ -0,0 +1,136 @@
+namespace Nancy.Tests.Unit
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    using Xunit;
+
+    public class HttpLinkHeaderFixture
+    {
+        private const string LinkHeader = "<http://nancyfx.org/>; "
+            + "rel=\"http://nancyfx.org/rel/Nancy\"; "
+            + "title=\"Nancy\", "
+            + "</alternate.jsonld>; "
+            + "rel=\"alternate\"; "
+            + "title=\"JSON LD Representation\"";
+
+

```

Comment:
All files are formatted according to the accompanying ReSharper `.DotSettings` file. Do you want me to tweak the settings in it so they don't add this whitespace?